install.sh

```diff
@@ -473,6 +484,11 @@ distro() {
     )
     return
   fi
+
+  if [ -f /etc/alpine-release ]; then
```

Comment:
It's too big of a change for this, I just wanted to mention that there's a standard file that contains distro info, `os-release`, which also seems to work on Alpine:

```
15:17 coder@jawnsy-m ~/projects/m $ docker run --rm -it fedora cat /etc/os-release | grep "^ID="
ID=fedora
15:17 coder@jawnsy-m ~/projects/m $ docker run --rm -it ubuntu cat /etc/os-release | grep "^ID="
ID=ubuntu
15:17 coder@jawnsy-m ~/projects/m $ docker run --rm -it debian cat /etc/os-release | grep "^ID="
ID=debian
15:17 coder@jawnsy-m ~/projects/m $ docker run --rm -it alpine cat /etc/os-release | grep "^ID="
ID=alpine
```

Here's the info about it: https://www.freedesktop.org/software/systemd/man/os-release.html

There's also a doc link earlier in the file here, do we want to use the new docs site instead? https://github.com/cdr/code-server/blob/5a6af177256056e316d0056b291fb3a8159c5776/install.sh#L70

Reply:
Ooo we use that a few lines up actually. @jsjoeio let's expand the case statement to include `alpine` instead of checking `/etc/alpine-release`. Actually I'm not sure why we need the case at all. Maybe let's just return whatever ID is in that file.

Reply:
Wait I misread. We only have the case for ID_LIKE then we fall back to ID. So this should already be returning `alpine` which means no changes are necessary here.

Reply:
> There's also a doc link earlier in the file here, do we want to use the new docs site instead?

Good catch! I'll update those.

Reply:
> I just wanted to mention that there's a standard file that contains distro info, os-release, which also seems to work on Alpine

TIL that's a thing! Thanks for the tip @jawnsy  🙌

> So this should already be returning alpine which means no changes are necessary in distro (can remove the /etc/alpine-release block).

Ah, didn't realize that. Sweet! Less code to add. Removed.